pom.xml

```diff
@@ -15,34 +15,31 @@
     <url>https://wiki.jenkins-ci.org/display/JENKINS/Extra+Tool+Installers+Plugin</url>
 
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <jenkins.version>1.651.3</jenkins.version>

```

Comment:
Going to jump all the way to 1.651.x but not just go to Jenkins 2?

Reply:
Yes for now. I am going to jump to 2.x once I am ready to update to parent POM 3.0 x